students/Gregg/lesson2/fizzbuzz.py

```diff
@@ -0,0 +1,46 @@
+def FizzBuzz(int_in):
```

Comment:
You'll want to make sure your function names pare PEP8 compliant - snake_case: def fizz_buzz()